README.md

```diff
@@ -1,182 +1,27 @@
-# Load Secrets from 1Password - GitHub Action
+<!-- Image sourced from https://blog.1password.com/1password-service-accounts/ -->
+<img alt="" role="img" src="https://blog.1password.com/posts/2023/1password-service-accounts/header.png"/>
 
-This action loads secrets from 1Password into GitHub Actions using [1Password Connect](https://developer.1password.com/docs/connect) or a [Service Account](https://developer.1password.com/docs/service-accounts).
+<header style="text-align: center;">
+  <h1 style="margin-top: 20px; border-bottom: none;">Load Secrets from 1Password - GitHub Action</h1>
+  <p>Provide the secrets your GitHub runner needs from 1Password.</p>
+  <a href="https://developer.1password.com/docs/ci-cd/github-actions">
+    <img alt="Get started" src="https://user-images.githubusercontent.com/45081667/226940040-16d3684b-60f4-4d95-adb2-5757a8f1bc15.png" height="37"/>
+  </a>
+</header>
```

Comment:
After merging in my `connect-sdk-js` PR I noticed that GitHub doesn't render these inline HTML `style` attributes correctly. I updated the template code in all of the issues to address this, but in short see below for the changes to make it render correctly.

```html
<div align="center">
  <h1>Load Secrets from 1Password - GitHub Action</h1>
  ...
</div>
```